extensions/data-plane/data-plane-http/src/main/java/org/eclipse/dataspaceconnector/dataplane/http/pipeline/HttpRequestParamsSupplier.java

```diff
@@ -66,10 +66,13 @@ public HttpRequestParams apply(DataFlowRequest request) {
                     params.contentType(ct);
                     params.body(extractBody(address, request));
                 });
+        Optional.ofNullable(extractTransferChunked(address)).ifPresent(params::transferChuncked);
```

Comment:
Our conventions are not to use Optionals like this

Reply:
I've refactor it to be more simple. Why you do not use optionals like this and how is the agreed usage within EDC (in order to align to the convention)?